proposals/dao/old/feiRari.ts

```diff
@@ -1,5 +1,5 @@
 import { expect } from 'chai';
-import { RunUpgradeFunc, SetupUpgradeFunc, TeardownUpgradeFunc, ValidateUpgradeFunc } from '../../types/types';
+import { RunUpgradeFunc, SetupUpgradeFunc, TeardownUpgradeFunc, ValidateUpgradeFunc } from '../../../types/types';
```

Comment:
If we're going to change paths we may as well switch to the @ syntax, which will likelyprevent the need for future path changes

Reply:
done